public/javascripts/handlebars-helpers.js

```diff
@@ -58,4 +58,14 @@ Handlebars.registerHelper('render_position', function(league, position){
     break;
   }
   return position;
+});
+
+Handlebars.registerHelper('format_height', function(heightInches){
```

Comment:
would be cool to eventually have this take an extra param for either murican/or non murican to display feet or meeters